samples/emoji-search/composeui/build.gradle

```diff
@@ -0,0 +1,79 @@
+apply plugin: 'com.android.application'
+apply plugin: 'org.jetbrains.kotlin.multiplatform'
+apply plugin: 'org.jetbrains.compose'
+
+redwoodBuild {
+  embedZiplineApplication(projects.samples.emojiSearch.presenterTreehouse)
+}
+
+android {
+  namespace 'com.example.redwood.emojisearch.composeui'
+
+  defaultConfig {
+    testInstrumentationRunner 'androidx.test.runner.AndroidJUnitRunner'
+  }
+
+  buildFeatures {
+    // Needed to pass application ID to UIAutomator tests.
+    buildConfig = true
+  }
+}
+
+compose {
+  desktop {
+    application {
+      mainClass = "com.example.redwood.emojisearch.composeui.Main"
+    }
+  }
+}
+
+kotlin {
+  androidTarget()
+  jvm("desktop")
```

Comment:
Not sure how I feel about these being in the same module. In the (admittedly very small) audience of those interested in Redwood and Treehouse, I expect the overwhelming majority to care about Android and not JVM Desktop. That's why I kept the counter frontends as separate modules so that you don't get lost in the multiplatform stuff around it. I would prefer to do the same here, unless there's some other strong reason to merge them.

Reply:
Sounds good I'll separate them 👍🏻